compiler/ballerina-compiler-api/src/main/java/module-info.java

```diff
@@ -1,4 +0,0 @@
-module org.ballerina.compiler {
-    requires io.ballerina.tools.api;
```

Comment:
With this we can directly access the BallerinaSemanticModel implementation as well as the SemanticModel API.

Reply:
Also, shall we add `exports io.ballerina.compiler.api.symbols;`, `exports io.ballerina.compiler.api.symbols;` and `exports io.ballerina.compiler.impl;` to the ballerina-lang's module-info?